### PR TITLE
PEP8 compliance

### PR DESCRIPTION
**Pull request by LightArrowsEXE**
Make a couple of changes to make vardefunc more PEP8 compliant (so it stops throwing me annoying errors). Only thing I didn't pay too much attention to was E501, as I do agree 79 characters per line is too little. Instead, I limited it to 120 characters (except for 1 line because there isn't a good break).

(Also, obviously gotta show my superiority :rampugun: :^))

pls2fix dis tho :^(
image

### CODE REVIEW

File: vardefunc.py [hunk 1]
```diff
@@ -3,26 +3,25 @@
...
 from functools import partial
+from typing import Any, Callable, Dict, Tuple, Union, cast
 
 import fvsfunc as fvf
 import havsfunc as hvf
-import placebo
-
-from vsutil import depth, get_depth, get_y, get_w, split, iterate
 import vapoursynth as vs
+from vsutil import depth, get_depth, get_w, get_y, iterate, split
```
Comment (Ichunjo) [inline vardefunc.py:12]: Why not order alphabetically here as well?

File: vardefunc.py [hunk 2]
```diff
@@ -3,26 +3,25 @@
...
 
-core = vs.core
+import placebo
```
Comment (Ichunjo) [inline vardefunc.py:14]: Why put it here?

File: vardefunc.py [hunk 3]
```diff
@@ -172,37 +174,44 @@ def nnedi3_upscale(clip: vs.VideoNode, scaler: Callable[[vs.VideoNode, Any], vs.
     return scaler(clip, src_top=.5, src_left=.5) if correct_shift else clip
 
 
-
 def fsrcnnx_upscale(source: vs.VideoNode, width: int = None, height: int = 1080, shader_file: str = None,
                     downscaler: Callable[[vs.VideoNode, int, int], vs.VideoNode] = core.resize.Bicubic,
-                    upscaler_smooth: Callable[[vs.VideoNode, Any], vs.VideoNode] = partial(nnedi3_upscale, nsize=4, nns=4, qual=2, pscrn=2),
+                    upscaler_smooth: Callable[[vs.VideoNode, Any], vs.VideoNode]
+                    = partial(nnedi3_upscale, nsize=4, nns=4, qual=2, pscrn=2),
```
Comment (Ichunjo) [inline vardefunc.py:180]: That’s something I wouldn't accept

File: vardefunc.py [hunk 4]
```diff
@@ -172,37 +174,44 @@ def nnedi3_upscale(clip: vs.VideoNode, scaler: Callable[[vs.VideoNode, Any], vs.
...
                     strength: float = 100.0, profile: str = 'slow',
                     lmode: int = 1, overshoot: int = None, undershoot: int = None,
-                    sharpener: Callable[[vs.VideoNode, Any], vs.VideoNode] = partial(z4USM, radius=2, strength=65))-> vs.VideoNode:
-    """Upscale the given luma source clip with FSRCNNX to a given width / height while preventing FSRCNNX artifacts by limiting them.
+                    sharpener: Callable[[vs.VideoNode, Any], vs.VideoNode]
```
Comment (Ichunjo) [inline vardefunc.py:183]: Same as L179

File: vardefunc.py [hunk 5]
```diff
@@ -251,7 +257,8 @@ def fsrcnnx_upscale(source: vs.VideoNode, width: int = None, height: int = 1080,
             limit = core.std.Expr([fsrcnnx, smooth], 'x y min')
         elif num == 2:
             # slow profile
-            upscaled = core.std.Expr([fsrcnnx, smooth], 'x {strength} * y 1 {strength} - * +'.format(strength=strength/100))
+            upscaled = core.std.Expr([fsrcnnx, smooth], 'x {strength} * y 1 {strength} - * +'
```
Comment (Ichunjo) [inline vardefunc.py:260]: This is ugly as fuck tho

File: vardefunc.py [hunk 6]
```diff
@@ -288,7 +292,11 @@ def fsrcnnx_upscale(source: vs.VideoNode, width: int = None, height: int = 1080,
     return out
 
 
-def to_444(clip: vs.VideoNode, width: int = None, height: int = None, join_planes: bool = True)-> vs.VideoNode:
+def to_444(clip: vs.VideoNode,
```
Comment (Ichunjo) [inline vardefunc.py:295]: Hell no

File: vardefunc.py [hunk 7]
```diff
@@ -633,7 +636,7 @@ def generate_keyframes(clip: vs.VideoNode, out_path: str) -> None:
     out_txt = ""
     for i in range(clip.num_frames):
         if clip.get_frame(i).props["_SceneChangePrev"] == 1 \
-            or clip.get_frame(i).props["Scenechange"] == 1:
+                or clip.get_frame(i).props["Scenechange"] == 1:
```
Comment (Ichunjo) [inline vardefunc.py:639]: ? 
flake8 doesn’t yell at you there

**Review by Ichunjo — CHANGES_REQUESTED** [review]

File: vardefunc.py (hunk 1 above)
Comment (LightArrowsEXE) [reply vardefunc.py:12]: That *is* alphabetically ordered 🤔

Comment (Ichunjo) [reply vardefunc.py:12]: ah yes indeed lmao

File: vardefunc.py (hunk 2 above)
Comment (LightArrowsEXE) [reply vardefunc.py:14]: It's a submodule from the same directory as vardefunc (as in it's included in this repo). It's handled similarly to how you'd import submodules into a `__init__` and separate those.

File: vardefunc.py (hunk 7 above)
Comment (Ichunjo) [reply vardefunc.py:633]: Ah I misread. I used flake8, not cornflakes

**Review by Ichunjo — APPROVED** [review]
Thanks Lighto 😘

**Pull request merged**